lib/hdiutil.js

```diff
@@ -22,14 +22,14 @@ exports.convert = function (source, format, target, cb) {
   })
 }
 
-exports.create = function (volname, size, cb) {
+exports.create = function (volname, size, filesystem, cb) {
   temp.template('%s.dmg').writeFile('', function (err, outname) {
     if (err) return cb(err)
 
     const args = [
       'create', outname,
       '-ov',
-      '-fs', 'HFS+',
+      '-fs', filesystem,
```

Comment:
I think that this needs to default to `'HFS+'`?

```suggestion
      '-fs', filesystem || 'HFS+',
```